README.md

```diff
@@ -20,11 +20,53 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install volabit
+    $ git clone https://github.com/coincovemx/api-client.git
+    $ cd api-client
+    $ bundle
+    $ gem build volabit.gemspec
+    $ gem install volabit-[version].gem
 
 ## Usage
 
-TODO: Write usage instructions here
+Instance a new Volabit client object.
+
+```ruby
+app_id      = 'The registered API for your APP.'
+secret      = 'The registered secret for your APP.'
+callback    = 'The registered callback URL for your APP'
+environment = 'The environment to work on (production or test)'
+
+client = Volabit::Client.new(app_id: app_id, app_secret: secret, app_url: callback, environment: environment)
+```
+
+Get the URL that will allow the user to authorize your app to use his/her account. (It should be opened in a browser.)
+
+```ruby
+auth_url = client.authorize
+```
+
+After you get the authorization code (sent at the callback URL that you provided), you'll use it to get the refresh and access tokens.
+
+```ruby
+client.get_token 'The given authorization code.'
```

Comment:
i think in the example is better to use `volabit_client` instead of just `client`. Some devs may already have client defined and it would collide.